src/odemis/gui/conf/file.py

```diff
@@ -205,6 +206,8 @@ def __init__(self):
         self.default.set("acquisition", "last_path", ACQUI_PATH)
         self.default.set("acquisition", "last_format", tiff.FORMAT)
         self.default.set("acquisition", "last_extension", tiff.EXTENSIONS[0])
+        self.default.set("acquisition", "fn_ptn", "%s" % self._get_default_filename)
+        self.default.set("acquisition", "fn_count", "0")
```

Comment:
Let's document here how the pattern and the count should be formatted. No need to have a full formal description, but enough info and links to other code in Odemis so that it's easy afterwards to read a pattern or write your own directly inside the config file.